.pre-commit-config.yaml

```diff
@@ -17,18 +17,18 @@ repos:
       - id: mixed-line-ending
       - id: trailing-whitespace
 
-  - repo: https://github.com/awebdeveloper/pre-commit-tslint/
-    rev: 'master'
-    hooks:
-      - id: tslint
-        files: \.(ts[x])$
-        # https://github.com/pre-commit/pre-commit/issues/1064
-        # https://github.com/ekalinin/nodeenv/issues/204
-        additional_dependencies:
-          - tslint
-          - tslint-react
-          - tslint-eslint-rules
-          - typescript
+  # - repo: https://github.com/awebdeveloper/pre-commit-tslint/
```

Comment:
tslint is deprecated and is broken both in ci and locally:


```
Run pre-commit run --all-files
[INFO] Initializing environment for pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/awebdeveloper/pre-commit-tslint/.
[INFO] Initializing environment for awebdeveloper/pre-commit-tslint/:tslint,tslint-eslint-rules,tslint-react,typescript.
[INFO] Initializing environment for pre-commit/mirrors-eslint.
[INFO] Initializing environment for pre-commit/mirrors-eslint:eslint@6.3.0.
[INFO] Installing environment for pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/awebdeveloper/pre-commit-tslint/.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: Command: ('/github/home/.cache/pre-commit/repowfj7e3z0/node_env-default/bin/node', '/github/home/.cache/pre-commit/repowfj7e3z0/node_env-default/bin/npm', 'install', '-g', '.', 'tslint', 'tslint-react', 'tslint-eslint-rules', 'typescript')
Return code: 1
Expected return code: 0
Output: (none)
Errors: 
    npm ERR! code ERESOLVE
    npm ERR! ERESOLVE unable to resolve dependency tree
    npm ERR! 
    npm ERR! While resolving: undefined@undefined
    npm ERR! Found: tslint@6.1.3
    npm ERR! node_modules/tslint
    npm ERR!   tslint@"*" from the root project
    npm ERR! 
    npm ERR! Could not resolve dependency:
    npm ERR! peer tslint@"^5.0.0" from tslint-eslint-rules@5.4.0
    npm ERR! node_modules/tslint-eslint-rules
    npm ERR!   tslint-eslint-rules@"*" from the root project
    npm ERR! 
    npm ERR! Fix the upstream dependency conflict, or retry
    npm ERR! this command with --force, or --legacy-peer-deps
    npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
    npm ERR! 
    npm ERR! See /github/home/.npm/eresolve-report.txt for a full report.
    
    npm ERR! A complete log of this run can be found in:
    npm ERR!     /github/home/.npm/_logs/2021-01-03T20_56_25_673Z-debug.log
    
Check the log at /github/home/.cache/pre-commit/pre-commit.log
```

Reply:
You can remove it for now from pre-commit and I'll work on moving to `eslint`